ntoskrnl/mm/ARM3/mdlsup.c

```diff
@@ -1622,29 +1622,228 @@ MmAdvanceMdl(IN PMDL Mdl,
 }
 
 /*
- * @unimplemented
+ * @implemented
  */
 PVOID
 NTAPI
 MmMapLockedPagesWithReservedMapping(IN PVOID MappingAddress,
                                     IN ULONG PoolTag,
```

Comment:
Some more annotations improvements (perhaps need to be improved too):
```suggestion
MmMapLockedPagesWithReservedMapping(_In_ PVOID MappingAddress,
                                    _In_ ULONG PoolTag,
```